packages/fxa-settings/src/components/InlineRecoveryKeySetupCreate/en.ftl

```diff
@@ -0,0 +1,8 @@
+## InlineRecoveryKeySetupCreate component
+
+inline-recovery-key-setup-signed-in-firefox = You’re signed in to { -brand-firefox }
+inline-recovery-key-setup-create-header = Secure your account
+inline-recovery-key-setup-create-subheader = Got a minute to protect your data?
```

Comment:
```suggestion
# This is a subheader asking users to create an account recovery key, indicating it will only take a moment to complete. 
# Possible alternatives: "Protect your data. It will only take a moment."
inline-recovery-key-setup-create-subheader = Got a minute to protect your data?
```
"Got a minute" is idiomatic and a bit colloquial which might make it hard for some languages to offer a natural localization here. It'd be nice to add a comment here will a more simplified alternative to assist localizers. (Suggestion above is just a suggestion, please feel free to adjust.)

Reply:
Nice catches on both. I've added that comment and for the missing strings, I meant to shuffle them over from a different component, so that's done. 👍